### PR TITLE
MEN-4625: update_control_maps integration tests

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: tests/tests/test_update_control.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        valid_image,
+    ):
+        """
+        Schedule an update with a pause in ArtifactInstall_Enter,
+        ArtifactReboot_Enter, and ArtifactCommit_Enter, then continue, after the
+        client has reported the paused substate back to the server, all the way
+        until the deployment is successfully finished.
+        """
+
+        u = User("", "bugs.bunny@acme.org", "whatsupdoc")
```
Comment (kacf) [inline tests/tests/test_update_control.py:43]: 🐰

**Review by kacf — APPROVED** [review]
Nice!

File: tests/tests/test_update_control.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            created_artifact = image.make_rootfs_artifact(
+                "../"+valid_image,
+                conftest.machine_name,
+                "test-update-control",
+                artifact_file.name,
+            )
+
+            deploy.upload_image(artifact_file.name, description="control map update test")
+
+        deployment_id = deploy.trigger_deployment(
```
Comment (kacf) [inline tests/tests/test_update_control.py:79]: This came up in the Slack channel: Can you, in addition to triggering the deployment, query the deployment afterwards, and make sure that the ID of the map it returns matches the ID of the deployment?

**Review by kacf — CHANGES_REQUESTED** [review]

File: tests/tests/test_update_control.py (hunk 2 above)
Comment (oleorhagen) [reply tests/tests/test_update_control.py:79]: Ref: https://northern-tech.slack.com/archives/C2D3HRU9M/p1623224323008800

Comment (oleorhagen) [reply tests/tests/test_update_control.py:79]: @kacf updated!

**Review by kacf — APPROVED** [review]
Great, thanks!

**Review by lluiscampos — APPROVED** [review]
I see there are still some debugging going on, but the test design looks good.

Efficient, clean, and very informative test. Thanks :+1:

**Pull request merged**